cvmfs/cvmfs.cc

```diff
@@ -1331,6 +1358,11 @@ static void cvmfs_getxattr(fuse_req_t req, fuse_ino_t ino, const char *name,
     attribute_value = string(VERSION) + "." + string(CVMFS_PATCH_LEVEL);
   } else if (attr == "user.pubkeys") {
     attribute_value = mount_point_->signature_mgr()->GetActivePubkeys();
+  } else if (attr == "user.repo_counters") {
+    attribute_value = mount_point_->catalog_mgr()->GetRootCatalog()->
+                                    GetCounters().GetCsvMap();
+  } else if (attr == "user.repo_metainfo") {
+    attribute_value = GetRepoMetainfo(size);
```

Comment:
It might be more complicated, we should get the hash of the meta info before `fuse_remounter_->fence()->Leave()`, but actually fetch the object only after leaving the fence.